src/tests/Equinor.ProCoSys.IPO.WebApi.IntegrationTests/Invitations/InvitationsControllerNegativeTests.cs

```diff
@@ -52,52 +56,82 @@ public async Task GetInvitation_AsViewer_ShouldReturnNotFound_WhenUnknownId()
                 HttpStatusCode.Unauthorized);
 
         [TestMethod]
         public async Task CreateInvitation_AsHacker_ShouldReturnBadRequest_WhenUnknownPlant()
             => await InvitationsControllerTestsHelper.CreateInvitationAsync(
-                AuthenticatedHackerClient(TestFactory.UnknownPlant),
+                UserType.Hacker,
+                TestFactory.UnknownPlant,
                 Guid.NewGuid().ToString(),
                 Guid.NewGuid().ToString(),
                 Guid.NewGuid().ToString(),
                 DisciplineType.DP,
+                _invitationStartTime, 
+                _invitationEndTime, 
+                _participants,
+                _mcPkgScope,
+                null,
                 HttpStatusCode.BadRequest,
                 "is not a valid plant");
 
         [TestMethod]
         public async Task CreateInvitation_AsHacker_ShouldReturnForbidden_WhenPermissionMissing()
             => await InvitationsControllerTestsHelper.CreateInvitationAsync(
-                AuthenticatedHackerClient(TestFactory.PlantWithAccess),
+                UserType.Hacker,
+                TestFactory.PlantWithAccess,
                 Guid.NewGuid().ToString(),
                 Guid.NewGuid().ToString(),
                 Guid.NewGuid().ToString(),
                 DisciplineType.DP,
+                _invitationStartTime,
+                _invitationEndTime,
+                _participants,
+                _mcPkgScope,
+                null,
                 HttpStatusCode.Forbidden);
 
         [TestMethod]
         public async Task CreateInvitation_AsSigner_ShouldReturnForbidden_WhenPermissionMissing()
             => await InvitationsControllerTestsHelper.CreateInvitationAsync(
-                SignerClient(TestFactory.PlantWithAccess),
+                UserType.Signer,
+                TestFactory.PlantWithoutAccess,
                 Guid.NewGuid().ToString(),
                 Guid.NewGuid().ToString(),
                 Guid.NewGuid().ToString(),
                 DisciplineType.DP,
+                _invitationStartTime,
+                _invitationEndTime,
+                _participants,
+                _mcPkgScope,
+                null,
                 HttpStatusCode.Forbidden);
 
         [TestMethod]
         public async Task CreateInvitation_AsViewer_ShouldReturnForbidden_WhenPermissionMissing()
             => await InvitationsControllerTestsHelper.CreateInvitationAsync(
-                ViewerClient(TestFactory.PlantWithAccess),
+                UserType.Viewer,
+                TestFactory.PlantWithoutAccess,
                 Guid.NewGuid().ToString(),
                 Guid.NewGuid().ToString(),
                 Guid.NewGuid().ToString(),
                 DisciplineType.DP,
+                _invitationStartTime,
+                _invitationEndTime,
+                _participants,
+                _mcPkgScope,
+                null,
                 HttpStatusCode.Forbidden);
 
         #endregion
```

Comment:
Will tests for edit (below) come in Vol2? Add a Todo

Reply:
Yes they will 👍  i'll add todo